shell.nix

```diff
@@ -12,7 +12,7 @@ verible_used_stdenv.mkDerivation {
   name = "verible-build-environment";
   buildInputs = with pkgs;
     [
-      bazel_5
+      bazel_7
       jdk11
```

Comment:
Maybe change this to bazel 6 as a good middle-ground.

That way, the user can test the 'old' form (without `--enable_bzlmod`) and the new.